tests/test_integrated_channels/test_sap_success_factors/test_exporters/test_content_metadata.py

```diff
@@ -303,11 +303,13 @@ def test_transform_courserun_title_excludes_start(self):
                         "key": "course-v1:CatalystX+IL-BSL.S1x+2T2017",
                         "start": "2017-05-16T16:00:00Z",
                         "end": "2017-06-30T23:59:00Z",
+                        "content_type": "courserun"
```

Comment:
From looking at some example data in production, I don't think this would be in the actual metadata for a course ContentMetadata item.